scripts/install.sh

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-sudo yum install python36
+sudo yum -y install python36
```

Comment:
I guess I missed this in e998f12f but I think think this line should be after `set -e` to force an exit if python36 doesn't install for some reason. Placing the line isn't really in the scope of this change (adding the '-y' arg) so I'm OK with addressing in later.